scripts/create_addon

```diff
@@ -91,6 +91,13 @@ pack_addon() {
   if [ ! -z "$PKG_ADDON_NAME" ] ; then
     ADDON_NAME="$PKG_ADDON_NAME"
   fi
+  for f in $PKG_DIR/repo/screenshot-*.{jpg,png}; do if [ -f $f ];
```

Comment:
yes there is a clash - we have already the `resources` folder for the `settings.xml` and languages, I thought this is rather messy to name it `resources` - also it copies the whole content of the `resources` folder to the addon.zip
there is no need to name it `repo` - i had simply no better idea (maybe `le_repo` ...)